File: Pythonleetcode/Leetcode66.py
class Solution(object):
    def plusOne(self, digits):
        n=0

        x=[]
        for i in range(0,len(digits)):
            n=n+digits[i]*(10**(len(digits)-i-1))
        s=str(n+1)
        for i in range(0,len(s)):
            x.append(int(s[i]))
        return x







print(Solution().plusOne([1,2,9]))

File: CppPractice/CppLeetcode/Leetcode66.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) 
    {
        reverse(digits.begin(),digits.end());
        vector<int> ans;
        int carry=0;
        digits[0]++;
        for(int i=0;i<digits.size();i++)
        {
            ans.push_back((carry+digits[i])%10);
            carry = (carry+digits[i])/10;
        }
        if(carry==1){ans.push_back(1);}
        reverse(ans.begin(),ans.end());
        return ans;

        
    }
};